integrations/terraform/protoc-gen-terraform-accesslist.yaml

```diff
@@ -71,6 +75,8 @@ validators:
   # Expires must be in the future
   Metadata.expires:
     - github_com_gravitational_teleport_integrations_terraform_tfschema.MustTimeBeInFuture()
+  AccessListAudit.next_audit_date:
+    - github_com_gravitational_teleport_integrations_terraform_tfschema.MustTimeBeInFuture()
```

Comment:
I think that this might break `terraform apply` when trying to update an access list whose audit is due. I would remove this validation as TF does not own the field, this is handled by Teleport.

Reply:
It's fine to create an initial next_audit_date during creation, but you're right. I'll remove this.

Reply:
https://github.com/gravitational/teleport/compare/900eef52e99002f42106b677b7773cf842e8b385..b15b9cd69ed7291b1a4f030ce6d05fe43bbc0647